data/build_redial.py

```diff
@@ -0,0 +1,156 @@
+# Copyright 2020 Google LLC
```

Comment:
Perhaps you are already doing this, but to check.

Can you please make sure you check that your files conform to the Google code style-guide using Pylint ?

Here are  instructions:
https://www.tensorflow.org/community/contribute/code_style?hl=ko
In internal Google code repository, this checks are done automatically during sumbit to ensure that there are no violations of the style guide.

Reply:
I was running regular pylint, but I switched to that version and fixed all the errors.